UPDATE.md

```diff
@@ -49,7 +49,7 @@ with open('css/materialdesignicons.css', 'r') as fp:
     rawcss = fp.read()
 
 charmap = {}
-pattern = '^\.mdi-(.+):before {\s*content: "(.+)";\s*}$'
+pattern = '^\.mdi-(.+)::before {\s*content: "(.+)";\s*}$'
```

Comment:
Can you give some context behind this change?

Reply:
On previous version the css looks like
```
.mdi-account-clock:before {
  content: "\FB32";
}
```

and on the last version (notice the double ::)
```
.mdi-account-clock::before {
  content: "\FB32";
}
```

Reply:
You could compare :
http://cdn.materialdesignicons.com/4.4.95/css/materialdesignicons.css

and 
http://cdn.materialdesignicons.com/3.8.95/css/materialdesignicons.css